### DISCUSSION TITLE
When i use getStaicPaths it return errorFetchError: invalid json response body at http://localhost:8000/BlogRead/next-js-blogs-l-2 reason: Unexpected token < in JSON at position 0

### DISCUSSION THREAD

**Discussion by akhilpamwani** [vercel/next.js#36546] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

{
  "name": "hunting-coder",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "aos": "^2.3.4",
    "axios": "^0.26.0",
    "next": "^12.1.5",
    "node-fetch": "^3.2.3",
    "react": "17.0.2",
    "react-dom": "17.0.2",
    "react-router-dom": "^6.3.0",
    "react-toastify": "^8.2.0",
    "sass": "^1.49.7",
    "sharp": "^0.30.1"
  },
  "devDependencies": {
    "autoprefixer": "^10.4.2",
    "eslint": "8.8.0",
    "eslint-config-next": "12.0.10",
    "postcss": "^8.4.6",
    "tailwindcss": "^3.0.22"
  }
}

### What browser are you using? (if relevant)

Microsoft  Edge

### How are you deploying your application? (if relevant)

Now it is under development

### Describe the Bug

when I use getStaticpath it returns this error   FetchError: invalid json response body at http://localhost:8000/BlogRead/next-js-blogs-l-2 reason: Unexpected token < in JSON at position 0

### Expected Behavior

FetchError: invalid json response body at http://localhost:8000/BlogRead/next-js-blogs-l-2 reason: Unexpected token < in JSON at position 0

### To Reproduce

1 Clone my repo = https://github.com/akhilpamwani/CodeHustler  && https://github.com/akhilpamwani/CodeHustler-Backend-
2 Install the requied packages using this command =  npm install
3 Start the Localhost Server by using this command  = yarn dev && yarn server
4 Go the url http://[localhost:3000/blogpost/next-js-blogs-l-2](http://localhost:3000/blogpost/next-js-blogs-l-2)
5 You find that error

**Comment by icyJoseph** [comment]
Hi,

Did you manage to solve this?

```js
  const res = await fetch(process.env.NEXT_PUBLIC_BLOG_READ_API_URL);
  const data = await res.json();
```

```js
  const res = await fetch(process.env.NEXT_PUBLIC_BLOG_SINGLE_API_URL+'/'+slug);

  const getData = await res.json();
```

These parts are worrying, because it looks like we assume that the request never fails. In this case it is failing, but it seems to me that it must be failing on the server.

From your browser, can you reach your server? Just paste the value of `process.env.NEXT_PUBLIC_BLOG_READ_API_URL`(http://localhost:8000/BlogRead/) or `NEXT_PUBLIC_BLOG_SINGLE_API_URL`(http://localhost:8000/BlogReadOne, with a slug) directly into the browser, can you see any data?

**Comment by akhilpamwani** [comment]
This problem is solved but there is another error arrived that is next/Babel modules not found while deploying to vercel

**Reply by icyJoseph** [reply]
Could you copy/paste, or take a screenshot of the actual error message?
